tools/automatic_query_fixer/gradlew

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
```

Comment:
Do we have to check in this? i forgot..

Reply:
I don't know. This is auto generated by the IDE.

Reply:
Decide to keep this observing other interns' repo.